### PR TITLE
Add agent uninstall RBAC action

### PR DESCRIPTION
**Pull request by GGP1**
## Description

Adds the `agent:uninstall` RBAC action.

### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: source/user-manual/api/rbac/reference.rst [hunk 1]
```diff
@@ -129,6 +129,11 @@ agent:upgrade
 -  :api-ref:`PUT /agents/upgrade <operation/api.controllers.agent_controller.put_upgrade_agents>` (:ref:`agent:id <api_rbac_reference_resources>`, :ref:`agent:group <api_rbac_reference_resources>`)
 -  :api-ref:`PUT /agents/upgrade_custom <operation/api.controllers.agent_controller.put_upgrade_custom_agents>` (:ref:`agent:id <api_rbac_reference_resources>`, :ref:`agent:group <api_rbac_reference_resources>`)
 
+agent:uninstall
+~~~~~~~~~~~~~~~
+
+-  :api-ref:`GET /agents/uninstall <operation/api.controllers.agent_controller.get_agent_uninstall_permission>` (:ref:`*:* <api_rbac_reference_resources>`)
```
Comment (GGP1) [inline source/user-manual/api/rbac/reference.rst:135]: [Reference](https://github.com/wazuh/wazuh/blob/b7dcc227e48732b77db2c92c1087e322a55d8ae9/api/api/spec/spec.yaml#L8279)

File: source/user-manual/api/rbac/reference.rst [hunk 2]
```diff
@@ -568,6 +573,7 @@ Grant full access to all agents related functionalities.
      actions:
        - agent:create
        - group:create
+       - agent:uninstall
```
Comment (GGP1) [inline source/user-manual/api/rbac/reference.rst:576]: [Reference](https://github.com/wazuh/wazuh/blob/b7dcc227e48732b77db2c92c1087e322a55d8ae9/framework/wazuh/rbac/default/policies.yaml)

**Review by RamosFe — APPROVED** [review]
LGTM.

**Review by fdalmaup — APPROVED** [review]
LGTM!

**Pull request merged**